pinot-common/src/test/java/org/apache/pinot/common/data/FieldSpecTest.java

```diff
@@ -24,7 +24,7 @@
 import java.util.Random;
 import java.util.concurrent.TimeUnit;
 import org.apache.avro.Schema;
-import org.apache.pinot.common.utils.AvroSchemaUtil;
+import org.apache.pinot.avro.data.readers.AvroSchemaUtil;
```

Comment:
we should try to get rid of this as well. Can you add it to the follow up task ticket?

Reply:
I don't understand.  Are you suggesting that we get rid of AvroSchemaUtil altogether?